src/index.html

```diff
@@ -4,9 +4,330 @@
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Alegreya&family=Alegreya+Sans&family=Archivo:wght@100;600;700&family=Fira+Sans:wght@300;400;500&family=Noto+Sans:wght@300;400;500;600;700&family=Open+Sans:wght@300&family=Oxygen:wght@400;700&family=Roboto:wght@100;300;400;500;700;900&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
+  <header class="header">
+    <a href="#home" class="header__logo">
+      <img
+        src="images/logo.png" 
+        alt="logo image"
+      >
+    </a>
+    <nav class="nav">
+      <ul class="nav__list">
+
+        <li class="nav__item">
+          <a href="#" class="nav__link is-active">Apple</a>
+        </li>
+
+        <li class="nav__item">
+          <a href="#" class="nav__link">Samsung</a>
+        </li>
+
+        <li class="nav__item">
+          <a href="#" class="nav__link">Smartphones</a>
+        </li>
+
+        <li class="nav__item">
+          <a href="#"
+            class="nav__link"
+            data-qa="nav-hover"
+          >
+            Laptops & Computers
+          </a>
+        </li>
+
+        <li class="nav__item">
+          <a href="#" class="nav__link">Gadgets</a>
+        </li>
+
+        <li class="nav__item">
+          <a href="#" class="nav__link">Tablets</a>
+        </li>
+
+        <li class="nav__item">
+          <a href="#" class="nav__link">Photo</a>
+        </li>
+
+        <li class="nav__item">
+          <a href="#" class="nav__link">Video</a>
+        </li>
+
+      </ul>
+    </nav>
+  </header>
   <body>
-    <h1>Catalog</h1>
+    <main class="main">
+      <section class="card main__card" data-qa="card">
+        <div class="card__image-container">
+          <img src="images/imac.jpeg" 
+            class="card__image" 
+
+
+            alt="APPLE A1419 iMac 27 Retina 5K Monoblock">
+        </div>
+  
+        <h2 class="card__name">APPLE A1419 iMac 27" Retina<br>5K Monoblock (MNED2UA/A)</h2>
+        <p class="card__code">Product code: 195434</p>
+  
+        <div class="card__rating">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="card__rewiews">Reviews: 5</div>
+        </div>
+  
+        <div class="card__price">
+          <p>Price:</p>
+          <p class="card__price-sum">$2,199</p>
+        </div>
+  
+        <a
+          href="#" 
+          class="card__button" 
+          data-qa="card-hover"
+        >
+          Buy
+        </a>
+      </section>
+      <section class="card main__card">
+        <div class="card__image-container">
+          <img src="images/imac.jpeg" 
+            class="card__image" 
+            alt="APPLE A1419 iMac 27 Retina 5K Monoblock">
```

Comment:
Pay attention on code style
```suggestion
          <img 
            src="images/imac.jpeg" 
            class="card__image" 
            alt="APPLE A1419 iMac 27 Retina 5K Monoblock"
          >
```

Reply:
isn't fixed yet